src/pages/List/TicketList.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import Ticket from "./Ticket";
+
+class TicketList extends Component {
+  render() {
+    const { ticket } = this.props;
+    console.log("props>>>", this.props.ticket);
+    return (
+      <div className="ticketList">
+        {ticket && (
+          <ul>
+            {ticket.map((el) => (
+              <Ticket
+                key={el.id}
+                // airlineIcon={el.airlineIcon}
```

Comment:
```suggestion
    return (
      <div className="ticketList">
          <ul>
            {ticket.length > 0 && ticket.map((el) => (
              <Ticket
                key={el.id}
                // airlineIcon={el.airlineIcon}
```
- ticket의 초기값은 []이고 빈 배열은 언제나 true로 계산됩니다. runJS에 직접 쳐보세요.
- 이 말인즉슨 ticket && 이라고 적어도 언제나 true인 조건문이 된다는 것이고, 이는 의미 없는 조건문이 된다는 뜻이기도 합니다. 따라서 배열의 length을 기준으로 조건부 렌더링 할 수 있도록 해주세요.

Reply:
수정했습니다.!!